packages/@office-iss/react-native-win32/src/Libraries/Text/TextProps.win32.js

```diff
@@ -0,0 +1,279 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {
+  AccessibilityActionEvent,
+  AccessibilityActionInfo,
+  AccessibilityRole,
+  AccessibilityState,
+  Role,
+} from '../Components/View/ViewAccessibility';
+import type {TextStyleProp} from '../StyleSheet/StyleSheet';
+import type {
+  LayoutEvent,
+  PointerEvent,
+  PressEvent,
+  TextLayoutEvent,
+} from '../Types/CoreEventTypes';
+import type {Node} from 'react';
+
+export type PressRetentionOffset = $ReadOnly<{|
+  top: number,
+  left: number,
+  bottom: number,
+  right: number,
+|}>;
+
+type PointerEventProps = $ReadOnly<{|
+  onPointerEnter?: (event: PointerEvent) => void,
+  onPointerLeave?: (event: PointerEvent) => void,
+  onPointerMove?: (event: PointerEvent) => void,
+|}>;
+
+/**
+ * @see https://reactnative.dev/docs/text#reference
+ */
+export type TextProps = $ReadOnly<{|
+  ...PointerEventProps,
+
+  /**
+   * Indicates whether the view is an accessibility element.
+   *
+   * See https://reactnative.dev/docs/text#accessible
+   */
+  accessible?: ?boolean,
+  accessibilityActions?: ?$ReadOnlyArray<AccessibilityActionInfo>,
+  onAccessibilityAction?: ?(event: AccessibilityActionEvent) => mixed,
+  accessibilityHint?: ?Stringish,
```

Comment:
What's Stringish?

Reply:
Good question.  Turns out for basically anything that matters, its a string.
```
// This type allows Facebook to internally Override
// this type to allow our internationalization type which
// is a string at runtime but Flow doesn't know that.
declare type Stringish = string;
```